### ISSUE TITLE
Failed to load URL: electron-app/out/main/index.html with error: ERR_FILE_NOT_FOUND

### ISSUE THREAD

**Issue by kipBO** [alex8088/electron-vite#138]
### Describe the bug

When I run start: `"start": "electron-vite preview"` I get the error `electron-app/out/main/index.html with error: ERR_FILE_NOT_FOUND`

It says index.html needs to be in main but the docs say it needs to be in renderer folder(which it is)
https://evite.netlify.app/guide/project-structure.html#recommended-project-structure

### Electron-Vite Version

1.0.21

### Electron Version

22.3.4

### Vite Version

-

### Validations

- [X] Follow the [Code of Conduct](https://github.com/alex8088/electron-vite/blob/master/CODE_OF_CONDUCT.md).
- [X] Read the [Contributing Guidelines](https://github.com/alex8088/electron-vite/blob/master/CONTRIBUTING.md).
- [X] Read the [docs](https://evite.netlify.app).
- [X] Check that there isn't [already an issue](https://github.com/alex8088/electron-vite/issues) that reports the same bug to avoid creating a duplicate.

**Comment by alex8088** [comment]
how did you config it?

**Comment by alex8088** [comment]
your project structure?

and `bytecodeplugin` not apply to renderer

**Comment by kipBO** [comment]
![image](https://user-images.githubusercontent.com/21161009/228926912-075bc9d8-45e2-4567-9807-ad036cc53c6e.png)
I'm using the default Recommended Project Structure trying to utilizing The default entry points.

**Comment by alex8088** [comment]
https://evite.netlify.app/guide/#scaffolding-your-first-electron-vite-project
```
npm create @quick-start/electron
```

**Comment by kipBO** [comment]
> your project structure?
> 
> and `bytecodeplugin` not apply to renderer
> 
> how did you load the `html`? https://evite.netlify.app/guide/hmr-in-renderer.html

Yep that was it, sorry I was working in the @quick-start/electron indeed.
But I had that in my project folder. I already had `mainWindow.loadURL(process.env['ELECTRON_RENDERER_URL'])` in my project but not in the quick-start folder where I was trying to get things to work. Thanks for the support!
